Application/EdFi.Ods.Api/Controllers/DataManagementControllerBase.cs

```diff
@@ -359,24 +359,6 @@ private EntityTagHeaderValue GetEtag(string etagValue)
             return new EntityTagHeaderValue(Quoted(etagValue));
         }
 
-        protected string GetResourceUrl()
```

Comment:
Moved this method to `HttpRequestExtensions` so that we can call it from the `IdentitiesController`.